tests/conftest.py

```diff
@@ -2938,3 +2943,28 @@ def smbios_from_kubevirt_config(kubevirt_config_scope_module):
 @pytest.fixture(scope="session")
 def nmstate_required(admin_client):
     return get_cluster_platform(admin_client=admin_client) in ("BareMetal", "OpenStack")
+
+
+@pytest.fixture(scope="package")
```

Comment:
I think "package" is too big, you have only one file with aaq here, so "module" should be enough